client/stylesheets/_responsive.scss

```diff
@@ -221,13 +257,23 @@
         margin: 8px 0 10px 0 !important;
         font-size: 1rem !important;
     }
+
     .searchResultsCardSize, .searchResultsCardCondition, .searchResultsCardLocation {
         margin: 12px 0 !important;
         font-size: 1rem !important;
     } 
+
     .searchResultsCardName {
         font-size: 0.95rem !important;
     }
+
+    .profileUserRating ul li span {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector `profileUserRating` should be written in lowercase with hyphens